### ISSUE TITLE
Loss not decreasing with PointLSTM with nuscenes

### ISSUE THREAD

**Issue by Jagyan** [hehefan/PointRNN-PyTorch#3]
Hi @hehefan,
This is an urgent request as I have a deadline to complete a project where I am using your network. I am running the model on nuscenes data and the loss is fluctuating within a certain range and not going down. I have already implemented the range of [-5m, 5m]. Is there any link to a prepared dataset? What else are you using for data preparation? I am using PointLSTM.

Regards.

![Training loss](https://user-images.githubusercontent.com/19373080/120579552-036ad100-c428-11eb-8c07-58e6f43ebe26.PNG)

**Comment by hehefan** [comment]
Hi @Jagyan,

I am sorry I have not applied this PyTorch version to nuscenes.  I am sure it effectively works on the Moving MNIST Point Cloud dataset. I would like to suggest you use the TensorFlow version. I will check the problem in the future. 

Another solution is to use only a few training samples, e.g., part of the test set, to check whether the loss goes down. 

Thank you.

**Comment by Jagyan** [comment]
I used a few training samples and could see some reduction in loss of the Lidar dataset of Nuscenes. My task is to implement it on radar points of nuscenes. However, the model is not able to learn anything from the data. I believe as the number of points per frame is just around 500 points as compared to 30k points of lidar, there is no learning happening. Do you have any suggestions on how to get results from the radar data? I have tried changing the loss function. But that did not work.

Thank you.

**Comment by hehefan** [comment]
Hi @Jagyan,

I think the radar points may be too sparse. Could you please visualize the radar points? If too sparse, it is difficult to capture the structure for modeling. 

Best regards.
